Makefile

```diff
@@ -31,9 +34,26 @@ MVN_COMMAND = mvn
 APB_COMMAND = docker run --rm --privileged -v `pwd`:/mnt -v ${HOME}/.kube:/.kube -v /var/run/docker.sock:/var/run/docker.sock -u `id -u` docker.io/ansibleplaybookbundle/apb
 
 _TEST_PROJECT = myproject
-_REGISTRY_IP = $(shell oc get svc/docker-registry -n default -o yaml | grep 'clusterIP:' | awk '{print $$2}')
-_IMAGE = $(_REGISTRY_IP):5000/$(_TEST_PROJECT)/$(DEV_IMAGE_NAME)
-_APB_IMAGE = $(_REGISTRY_IP):5000/$(_TEST_PROJECT)/$(DEV_APB_IMAGE_NAME)
+
+#Change addresses for remote OpenShift only when _OPENSHIFT_DOMAIN env variable is set.
+ifeq ($(_OPENSHIFT_DOMAIN),)
+	_DOCKER_REGISTRY = "$(shell oc get svc/docker-registry -n default -o yaml | grep 'clusterIP:' | awk '{print $$2}'):5000"
+	_IMAGE = $(_DOCKER_REGISTRY)/$(_TEST_PROJECT)/$(DEV_IMAGE_NAME)
+	_APB_IMAGE = $(_DOCKER_REGISTRY):5000/$(_TEST_PROJECT)/$(DEV_APB_IMAGE_NAME)
+	_USERNAME = developer
+	_PASSWORD = developer
+	_TESTRUNNER_HOST = testrunner-$(_TEST_PROJECT).127.0.0.1.nip.io
+	_TESTRUNNER_PORT = 80
+else
+	_OPENSHIFT_MASTER = https://osemaster.$(_OPENSHIFT_DOMAIN):8443
```

Comment:
You should probably take fill url as a parameter here. We might aim different clusters later on.

Reply:
Can you suggest how to do that? I need the OPENSHIFT_DOMAIN for Arquillian (the _TESTRUNNER_HOST) and I need OPENSHIFT_MASTER somewhere else. So I constructed OPENSHIFT_MASTER from OPENSHIFT_DOMAIN, the user only has to specify OPENSHIFT_DOMAIN and it's done. Do you suggest that the user would have to specify OPENSHIFT_DOMAIN and OPENSHIFT_MASTER separately? Or even _TESTRUNNER_HOST separately? I'm curious